src/chrome/content/rules/heritage.gov.hk.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	Problematic subdomains:
+		- (?:www\.)	(cert mismatch: www.heritage.gov.hk) 
```

Comment:
I would suggest you to list them like:
```
	Mismatch:
		- www.heritage.gov.hk
```
(Change the spaces to tabs, GitHub automatically transforms them in comments)